app/controllers/concerns/get_into_teaching_redirect.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module GetIntoTeachingRedirect
+  extend ActiveSupport::Concern
+
+  def git_redirect
+    redirect_to I18n.t("get_into_teaching.#{git_url_locale_key}"), allow_other_host: true
+  end
+
+  private
+
+  def git_path_param
+    params[:git_path]
```

Comment:
should we validate the params we accept?

If we put something silly as the param where do we get redirected to? i.e. where would `course/:provider_code/:course_code/git/help` redirect to?

Reply:
That will fail, but I think that's ok. It should fail if we don't have a locale. I thought about catching the failure and directing to the git home page, but that would hide the error.